introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-benchmarking/inference-benchmarking-custom-probe-example.ipynb

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.            self.max_latency_seconds = 25.
</pre><p>Why 25? Is it based on the queuing theory inflection point?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4443/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>